cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt

```diff
@@ -6,6 +6,7 @@ import java.util.*
 
 private val HEADLESS_FLAG = "--headless"
 private val CAPSULE_DEBUG_FLAG = "--capsule-debug"
+private val JOLOKIA_FLAG = "--jolokia"
```

Comment:
Could you make this "--expose-jolokia" to make it more obvious to those who may want to understand what a particular flat does?